feedback/models.py

```diff
@@ -306,6 +351,34 @@ def get_notresponded(self, exercise_id=None, course_id=None, path_filter=None):
 ResponseUploaded = namedtuple('ResponseUploaded',
                               ('ok', 'when', 'code', 'attempts'))
 
+class Conversation(models.Model):
+
+    class Meta:
+        unique_together = [
+            ('exercise', 'student')
+        ]
```

Comment:
Note, `unique_together` is likely to become deprecated. The newer [UniqueConstraint](https://docs.djangoproject.com/en/4.2/ref/models/constraints/#django.db.models.UniqueConstraint) from the [constraints](https://docs.djangoproject.com/en/4.2/ref/models/options/#django.db.models.Options.constraints) option is now recommended.

https://docs.djangoproject.com/en/4.2/ref/models/options/#unique-together